### DISCUSSION TITLE
Renomear Equipes

### DISCUSSION THREAD

**Discussion by VictorHugoNeivaDosSantos** [community/community#141386] [Code Security]
### Select Topic Area

Question

### Body

Não é póssivel renomear um nome de uma equipe criada ?
![image](https://github.com/user-attachments/assets/7962615a-b4c1-4ddb-8bb4-f55f834adbbd)

**Comment by ghostinhershell** [comment]
Hi @VictorHugoNeivaDosSantos, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
